lib/need_for_type/states/menu_state.rb

```diff
@@ -0,0 +1,20 @@
+module NeedForType::States
+  class MenuState < State
```

Comment:
My fault here, I wrote `MenuState` but as we said on the previous comments, we don't need the `State` portion, because of the namespace, so I think `Menu` would be enough here.